### PR TITLE
feat(content): #193 improve icons

### LINKED ISSUE #193: Improve ActivityStream icons
Running the latest code from master (0e6e8f7) and I just noticed that all my ActivityFeed items always use the fallback icon instead of the site's favicon (which is used in Top Sites, and Spotlight results).

new_tab

### LINKED ISSUE #286: icon displays wrong character on some articles
1. goto: https://news.ycombinator.com/item?id=11230084
2. open new tab and look at icon in history

actual:
screen shot 2016-03-07 at 11 54 23 am

expected: should be the letter 'N' or 'Y' for subdomain or domain.

### PR DESCRIPTION
**Pull request by k88hudson**
Fixes #193 
Fixes #286

### CODE REVIEW

File: content-src/components/SiteIcon/SiteIcon.scss [hunk 1]
```diff
@@ -1,23 +1,50 @@
+
+
```
Comment (pdehaan) [inline content-src/components/SiteIcon/SiteIcon.scss]: nit: extra whitespace

**Comment by pdehaan** [comment]
Here's what it looks like locally for me:

new_tab

---

new_tab

**Comment by k88hudson** [comment]
@rlr R?

File: content-src/lib/utils.js [hunk 2]
```diff
@@ -50,5 +74,10 @@ module.exports = {
     }
 
     return parsedUrl.toString();
+  },
+
+  getRandomFromTimestamp(percent, site) {
+    const n = (site.lastVisitDate || site.bookmarkDateCreated || Math.round(Math.random() * 100)) % 100;
```
Comment (rlr) [inline content-src/lib/utils.js]: nice

**Comment by rlr** [comment]
code LGTM and I played with it a bit and it works as expected. I think we might end up adding the background to spotlight images because they don't always bleed in well with the background. For example: 
screen shot 2016-03-14 at 4 10 57 pm

R+

**Pull request merged**